packages/features/bookings/lib/getBookingResponsesSchema.ts

```
@@ -103,7 +108,9 @@ function preprocess<T extends z.ZodType>({
           };
           try {
             parsedValue = JSON.parse(value);
-          } catch (e) {}
+          } catch {
+            // If parsing fails, we don't add the field to newResponses
```

Empty catch blocks is a bad practice, catch the error and throw it.